serpentTools/parsers/sensitivity.py

```diff
@@ -331,6 +331,10 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
         mat: None or str or list of strings
             Plot sensitivities due to these materials. Passing ``None``
             will plot against all materials.
+        mevscale: string
```

Comment:
Probably can just leave this as a boolean
```suggestion
        mevscale : bool, optional
```